src/build/cargo.rs

```diff
@@ -39,8 +39,8 @@ pub(super) fn cargo(internals: &Internals) -> BuildResult {
 
     let diagnostics = Arc::new(Mutex::new(vec![]));
     let diagnostics_clone = diagnostics.clone();
-    let analyses = Arc::new(Mutex::new(vec![]));
```

Comment:
These local variables should all be `analysis` not `analyzes`

Reply:
Thanks, done